src/UnitTests/Constructors.cs

```diff
@@ -8,6 +8,18 @@
 
 namespace AutoMapper.UnitTests.Constructors
 {
+    public class ConstructorValidation : AutoMapperSpecBase
+    {
+        class Source
+        {
+        }
+        class Destination
+        {
+            public Destination(int value = 2) => Value = value;
```

Comment:
People want to say that the property `Value` is mapped because this constructor is used.